common/ethnicities/wc_ethnicities_elven.txt

```diff
@@ -368,6 +374,114 @@ night_elven = {
 	gene_bs_ear_lenght = {
         10 = { name = floppy_ear_length    range = { 0.75 1 } }
     }
+	gene_bs_ear_angle = {
+        1 = { name = ear_angle_neg    range = { 0.6 1.0 }    }
+    }
+    gene_bs_ear_inner_shape = {
+        40 = { name = ear_inner_shape_pos    range = { 0.0 0.0 }    }
+	}
+    gene_bs_ear_outward = {
+        1 = { name = ear_outward_neg    range = { 0.7 1.0 }    }
+	}
+	gene_bs_ear_size = {
+        1 = { name = ear_size_neg    range = { 0.6 1.0 }    }
+	}
+	#Noses
+	gene_bs_nose_forward = {
+        1 = { name = nose_forward_pos    range = { 0.3 0.7   }    }
+	}
+	gene_bs_nose_length = {
+        1 = { name = nose_length_neg    range = { 0.0 0.3 }    }
+	}
+	gene_bs_nose_profile = {
+	    2 = { name = nose_profile_neg    range = { 0.1 0.4   }    }
+	}
+	gene_bs_nose_ridge_angle = {
+        1 = { name = nose_ridge_angle_pos    range = { 0.0 0.3   }    }
+	}
+	gene_bs_nose_ridge_width = {
+        1 = { name = nose_ridge_width_pos    range = { 0.0 0.3 }    }
+	}
+	gene_bs_nose_size = {
+        1 = { name = nose_size_pos    range = { 0.0 0.4 }    }
+	}
+    gene_bs_nose_tip_width = {
+        5 = { name = nose_tip_width_neg    range = { 0.8 1.0 }    }
+	}
+	#Cheeks
+	face_detail_cheek_def = {
+        10 = {  name = cheek_def_01             range = { 0.2 0.4 }     }
+	    10 = {  name = cheek_def_02             range = { 0.2 0.4 }     }
+	}
+	gene_bs_cheek_forward = {
+        1 = { name = cheek_forward_pos    range = { 0.6 1.0 }    }
+	}
+	gene_bs_cheek_height = {
+        1 = { name = cheek_height_neg    range = { 0.0 0.3 }    }
+	}
+	gene_bs_cheek_width = {
+        1 = { name = cheek_width_neg    range = { 0.0 0.4 }    }
+	}
+	#Chin
+	gene_chin_forward = {
+        1 = { name = chin_forward_neg    range = { 0.5 0.6 }    }
+	}
+	gene_chin_height = {
+        1 = { name = chin_height_neg    range = { 0.4 0.45 }    }
+	}
```

Comment:
This gene almost unchanged compared to the high elven portraits. Are you sure it's needed?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105350388-853e2880-5c04-11eb-8ae4-006c73292f85.png)

</details>
Also, check other barely changed genes. Sometimes it's very confusing: like I don't see what is really important in their look and what makes them different.

Reply:
They are just a small change to make the NE slightly different from the HE so that all of them don look the same, and some in some genes .05 is making some difference more than a whole value from what I could see while tweeking it